db/migrate/20190716121718_add_password_digest_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPasswordDigestToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :password_digest, :string
```

Comment:
@shaniajubelag
Let's not include this yet, since this is not related to this task.